### ISSUE TITLE
Feature Request: Action Relationship

### ISSUE THREAD

**Issue by fi0** [hasura/graphql-engine#6557]
Action relationship can be created  from an action to a local table.
Please also add support for creating a relationship from a local table to an action.

For instance,
An action accepts `user_id` as the input and return the `first_name`.
A local table includes a list of `user_id`. 
And I'd like hasura to return a list of users that contain both `user_id` and `first_name`.

**Comment by esseswann** [comment]
This functionality is available through Remote Schemas

**Comment by fi0** [comment]
It would be nice if non-GraphQL endpoints are also supported.
Or do you recommend using Remote Schemas with GraphQL in this case?

**Comment by esseswann** [comment]
Yes, you can create a relationship from your Hasura table to a Remote schema and it will automatically map corresponding entries
https://hasura.io/blog/remote-joins-a-graphql-api-to-join-database-and-other-data-sources/
